ci/meson/build_config.py

```
@@ -83,6 +105,89 @@ def _ensure_native_launcher(project_root: Path) -> tuple[Optional[str], Optional
+
+    try:
+        from clang_tool_chain.commands.compile_native import compile_native
+
+        output_dir.mkdir(parents=True, exist_ok=True)
+        rc = compile_native(str(output_dir))
+        if rc == 0 and ctc_emcc.exists() and ctc_empp.exists():
+            return str(ctc_emcc), str(ctc_empp)
+    except KeyboardInterrupt as ki:
+        handle_keyboard_interrupt(ki)
+    except Exception:
+        pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)